sanity/keylime-secure_mount/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+SECURE_DIR="/var/lib/keylime/secure"
+KEYLIME_UNIT_FILE="/etc/systemd/system/keylime_agent.service"
+if [ ! -f  "$KEYLIME_UNIT_FILE" ]; then
+    KEYLIME_UNIT_FILE="/usr/lib/systemd/system/keylime_agent.service"
+fi
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+
+        # update /etc/keylime.conf
+        limeBackupConfig
+        rlFileBackup $KEYLIME_UNIT_FILE
+        rlRun "limeUpdateConf logger_root level DEBUG"
+        rlRun "limeUpdateConf logger_keylime level DEBUG"
+        rlRun "limeUpdateConf handler_consoleHandler level DEBUG"
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify that agent creates mount point when not present and mounts tmpfs."
+        if [[ $(findmnt -M "$SECURE_DIR") ]]; then
```

Comment:
Nice, I didn't know this command.

Reply:
However, can't you simply utilize the exit code and simplify it to `findmnt -M "$SECURE_DIR" && umount "$SECURE_DIR"` ?